pkg/apiserver/handlers/webhook/convert_crd.go

```diff
@@ -1,240 +0,0 @@
-// Copyright 2021 Antrea Authors
```

Comment:
Should we keep this util code even though it's not used for now? It's likely required in the future and can save some code review effort when adding them back. @antoninbas

Reply:
I'm ok with both ways.

Reply:
+1, I feel we may still need this conversion webhook in the future.

Reply:
Added a commit reverting this deletion. Also, revert the v1alpha2 conversion for the "recoverable" story mentioned by Quan here: https://github.com/antrea-io/antrea/pull/5181#discussion_r1266916232

Reply:
yes, let's keep the code. I spent a lot of time recently reviewing the unit tests for it :)